scripts/debugCommands/cat.py

```diff
@@ -0,0 +1,83 @@
+from scripts.debugCommands.command import Command
+
+from scripts.debugCommands.utils import add_output_line_to_log
+
+from scripts.game_structure.game_essentials import game
+from scripts.cat.cats import Cat
+
+class addCatCommand(Command):
+    name = "add"
+    description = "Add a cat"
+    aliases = ["a"]
+
+    def callback(self, args: list[str]):
+        cat = Cat()
+        game.clan.add_cat(cat)
+        add_output_line_to_log(f"Added {cat.name} with ID {cat.ID}")
+
+class removeCatCommand(Command):
+    name = "remove"
+    description = "Remove a cat"
+    aliases = ["r"]
+    usage = "<cat name|id>"
+
+    def callback(self, args: list[str]):
```

Comment:
This type-hint syntax only works for python 3.9 and above, and causes crashes on the 3.8 builds.  I fixed it on the currently-existing debug commands here (https://github.com/Thlumyn/clangen/pull/1720).